### PR TITLE
rewrite all anko based eval statements into new eval.

### LINKED ISSUE #143: Old eval style
There are still some kits that use the old eval syntax, clean those out ASAP

### PR DESCRIPTION
**Pull request by david-fritz-gravwell**
Update versions on related kits.

Fixes #143

### CODE REVIEW

File: ipmi/MANIFEST [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"ID": "io.gravwell.ipmi",
 	"Name": "IPMI",
 	"Desc": "An IPMI kit that pairs with the Gravwell IPMI ingester.",
-	"Version": 4,
+	"Version": 5,
 	"MinVersion": {
 		"Major": 4,
```
Comment (kris-watts-gravwell) [inline ipmi/MANIFEST:7]: bump min version

File: linux_syslog/MANIFEST [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	"Name": "Linux Syslog",
 	"Desc": "Provides tools for working with Linux system logs, including ssh, sudo, and cron.",
 	"Readme": "# Linux Syslog Kit\r\n\r\nThis kit provides useful tools for working with system logs from Linux machines. It includes dashboards, queries, actionables, and templates built around logs from the SSH daemon, the cron daemon, and the sudo system.\r\n\r\nWe recommend installing the plain syslog kit (io.gravwell.syslog) as well, but this kit does *not* depend on it.",
-	"Version": 6,
+	"Version": 7,
 	"MinVersion": {
 		"Major": 5,
```
Comment (kris-watts-gravwell) [inline linux_syslog/MANIFEST:8]: bump min version

File: netflowv5/MANIFEST [hunk 3]
```diff
@@ -2,7 +2,7 @@
 	"ID": "io.gravwell.netflowv5",
 	"Name": "Netflow v5",
 	"Desc": "This kit provides ready-to-roll dashboards, queries, templates, playbooks, and actionables for analyzing Netflow v5",
-	"Version": 8,
+	"Version": 9,
 	"MinVersion": {
 		"Major": 4,
```
Comment (kris-watts-gravwell) [inline netflowv5/MANIFEST:7]: bump min version

File: paloalto/MANIFEST [hunk 4]
```diff
@@ -3,7 +3,7 @@
 	"Name": "Palo Alto",
 	"Desc": "Analyze Palo Alto logs.",
 	"Readme": "# Palo Alto Kit\n\nThis kit provides pre-built tools to help analyze logs from Palo Alto next-gen firewalls. It includes:\n\n* Autoextractor definitions for common Palo Alto log formats, to more easily extract fields from log entries.\n* Dashboards providing overviews \u0026 investigative options for different log types.\n* Instructions on how to get Palo Alto logs *into* Gravwell.",
-	"Version": 4,
+	"Version": 5,
 	"MinVersion": {
 		"Major": 5,
```
Comment (kris-watts-gravwell) [inline paloalto/MANIFEST:8]: bump min version

File: pfSense/MANIFEST [hunk 5]
```diff
@@ -3,7 +3,7 @@
 	"Name": "Gravwell for pfSense® Software",
 	"Desc": "Provides tools for working with pfSense® Software logs, including support for firewall logs.\n",
 	"Readme": "",
-	"Version": 1,
+	"Version": 2,
 	"MinVersion": {
 		"Major": 5,
```
Comment (kris-watts-gravwell) [inline pfSense/MANIFEST:8]: min version bump

File: pfSense/template/5c1740d8-2caa-416b-98ae-37da4e89c500.query [hunk 6]
```diff
@@ -1,7 +1,7 @@
 tag=pfsensesyslog regex "\<(?P<pri>\d+)\>(?P<timestamp>\w+\s+\d{1,2}\s+\d{2}:\d{2}:\d{2})\s+(?P<app_name>[\/\w]+)(:\s+)?(\[(?P<procid>\d+(:\d+)?)\]\s+)?(?P<msgid>.*)"
 | regex "%%SEARCHTERM%%"
-| eval setEnum("facility_code", toInt(pri)>>3)
-| eval setEnum("severity_code", toInt(pri)&7)
+| eval facility_code = int(pri)>>3;
```
Comment (kris-watts-gravwell) [inline pfSense/template/5c1740d8-2caa-416b-98ae-37da4e89c500.query:3]: condense into a single statement

File: pfSense/template/b6e9c6ff-97e4-4e66-a82e-4da1838fb9ef.query [hunk 7]
```diff
@@ -1,9 +1,9 @@
 tag=pfsensesyslog regex "\<(?P<pri>\d+)\>(?P<timestamp>\w+\s+\d{1,2}\s+\d{2}:\d{2}:\d{2})\s+(?P<app_name>[\/\w]+)(:\s+)?(\[(?P<procid>\d+(:\d+)?)\]\s+)?(?P<msgid>.*)"
 | regex "%%SEARCHTERM%%"
-| eval setEnum("severity", toInt(pri)&7)
+| eval severity = int(pri)&7;
 | eval severity <= 4
-| eval setEnum("facility_code", toInt(pri)>>3)
-| eval setEnum("severity_code", severity)
+| eval facility_code = int(pri)>>3;
+| eval severity_code = severity;
```
Comment (kris-watts-gravwell) [inline pfSense/template/b6e9c6ff-97e4-4e66-a82e-4da1838fb9ef.query:6]: condense into a single eval statement

File: pfSense/template/dfeee681-da79-44a1-9874-7908124a6802.query [hunk 8]
```diff
@@ -1,6 +1,6 @@
 tag=pfsensesyslog regex "\<(?P<pri>\d+)\>(?P<timestamp>\w+\s+\d{1,2}\s+\d{2}:\d{2}:\d{2})\s+(?P<app_name>[\/\w]+)(:\s+)?(\[(?P<procid>\d+(:\d+)?)\]\s+)?(?P<msgid>.*)"
 | regex "%%SEARCHTERM%%"
-| eval setEnum("facility", toInt(pri)>>3)
+| eval facility int(pri)>>3;
```
Comment (kris-watts-gravwell) [inline pfSense/template/dfeee681-da79-44a1-9874-7908124a6802.query:3]: shouldn't this have an assignment of some sort?

File: sysmon/MANIFEST [hunk 9]
```diff
@@ -3,7 +3,7 @@
 	"Name": "Windows Sysmon",
 	"Desc": "The Gravwell Sysmon kit provides a wide array of queries, dashboards, templates, and actionables to support monitoring and investigating Sysmon data.  The sysmon kit helps you monitor DNS, network, file, and registry activity provided by the Sysmon toolkit, it can be an invaluable resource for day-to-day monitoring as well as hunting misbehaving applications and malware.",
 	"Readme": "",
-	"Version": 4,
+	"Version": 5,
 	"MinVersion": {
```
Comment (kris-watts-gravwell) [inline sysmon/MANIFEST:7]: min version bump

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
lots of kit min version bumps and a few questions on eval syntax

**Comment by david-fritz-gravwell** [comment]
@kris-watts-gravwell PTAL

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**